src/sentry/api/bases/group.py

```diff
@@ -54,6 +54,9 @@ def convert_args(self, request, issue_id, *args, **kwargs):
             }
         )
 
+        if group.status in (GroupStatus.PENDING_DELETION, GroupStatus.DELETION_IN_PROGRESS):
```

Comment:
yeah good call. i wasn't sure what exactly that meant, but looks like it's applied to all but the merged-into group (see [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_group_index.py#L814)) so i think it makes sense to exclude that status also